src/rollup/index.ts

```diff
@@ -379,10 +366,12 @@ export default function rollup(
 					return promise;
 				}
 
-				const result: OutputChunk = {
-					imports: chunk.getImportIds(),
-					exports: chunk.getExportNames(),
-					modules: chunk.getJsonModules(),
```

Comment:
Just for curiosity: What is the reason you switched `modules` to plain ids instead of complex objects with meta information? Performance?
I am definitely ok with this. Should there actually be demand for this additional information in the future, one idea for the interface might be to create additional meta-information on-demand by providing a getter here instead of a pre-made object. This could even be done in the single-chunk case in a somewhat downward compatible way by using actual getters.

Reply:
The main reason is simply reducing "encouraged" API surface area. This is also why I created a new `cache` property to contain the full modules object on the Rollup output - so that users start to think of that ModuleJSON as something more internal to the cache than directly user-facing.